### PR TITLE
Updated SBCL Plugin to Github Actions

### PR DESCRIPTION
**Pull request by smashedtoatoms**
Updated the badge for SBCL from Travis to GithubActions.

# Checklist

- [x ] Provide a short description and link to your plugin.

- [ x] Make sure you CI tests for your plugin and they are green.

      If you are using github, you might want to use the
      `plugin_test` action from [asdf-actions](https://github.com/asdf-vm/actions)

- [x ] `asdf-plugins` CI sanity checks are green on your PullRequest.

      You can test locally using:

      ```bash
      ./test_plugin.sh --file plugins/PLUGIN_FILE
      ```

## Other Information

If there is anything else that is relevant to your pull request include that
information here.

Thank you for contributing to asdf-plugins!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -265,7 +265,7 @@ itself) to look for plugins.
...
 | starship                      | [grimoh/asdf-starship](https://github.com/grimoh/asdf-starship)                                       | [![GitHub Actions Status](https://github.com/grimoh/asdf-starship/workflows/Main%20workflow/badge.svg?branch=master)](https://github.com/grimoh/asdf-starship/actions)                                           |
-| Steel Bank Common Lisp (sbcl) | [smashedtoatoms/asdf-sbcl](https://github.com/smashedtoatoms/asdf-sbcl)                               | [![Build Status](https://travis-ci.org/smashedtoatoms/asdf-sbcl.svg?branch=master)](https://travis-ci.org/smashedtoatoms/asdf-sbcl)                                                                              |
+| Steel Bank Common Lisp (sbcl) | [smashedtoatoms/asdf-sbcl](https://github.com/smashedtoatoms/asdf-sbcl)                               | [![CI](https://github.com/smashedtoatoms/asdf-sbcl/workflows/CI/badge.svg?branch=master)](https://github.com/smashedtoatoms/asdf-sbcl/actions)                                                        |
```
Comment (smorimoto) [inline README.md:268]: Isn't the URL `Build`, not `CI`?

Comment (smashedtoatoms) [reply README.md:268]: I went off of the ones immediately around mine, which don't appear to be standardized.  I will set it to `Build`.  Sorry about that.

Comment (smorimoto) [reply README.md:268]: No need to apologize! By the way, it depends on the workflow name.

Comment (smashedtoatoms) [reply README.md]: I think I got it sorted, but it looks like everything is a little different.  I am not sure there is a standard anymore.

**Review by smorimoto — APPROVED** [review]

**Comment by smorimoto** [comment]
Looks good! Thanks! By the way, now with the latest `asdf-vm/actions`, we no longer need to set the `GITHUB_API_TOKEN` environment variable, so you can remove it from your workflow file!
https://github.com/asdf-vm/actions/releases/tag/v1.1.0

**Comment by smashedtoatoms** [comment]
That's good to know, I am removing it now!  Great work on the github actions,  I was expecting it to take hours to get working, and it took 10 minutes.  It is so nice to have all the hard work done for me.

**Comment by smorimoto** [comment]
I'm glad to hear that :)

**Pull request merged**